### ISSUE TITLE
Cloudflare CAPTCHA not able to be passed

### ISSUE THREAD

**Issue by candroid-man** [brave/brave-browser#40588]
### Description

I am unable to solve Cloudflare CAPTCHA on multiple websites including the official Cloudflare website. The video below shows me using Brave at first to try to solve the CAPTCHA, and then switching to Chromium on the same site and solving it no problem.

https://github.com/user-attachments/assets/36c43fce-e022-42cd-b72d-c043080d7ade

### Steps to reproduce

1. Find a website with a Cloudflare CAPTCHA
2. Try to solve it
3. See error

### Actual result

See video in description

### Expected result

See video in description. It is shown being solved in Chromium

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

**Brave:** 1.68.141 Chromium: 127.0.6533.120 (Official Build) unknown (64-bit)
**Revision:** b3f23500b575d50584510ea1814ef440d30741a8
**OS:** Linux

Downloaded from Flathub on Fedora Linux

### Channel information

- [X] release (stable)
- [ ] beta
- [ ] nightly

### Reproducibility

- [X] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

I am using custom DNS via ControlD and also using a VPN. I have disabled both of them and disabled shields, and no pertinent URLs are being blocked while ControlD is enabled.

**Comment by ervgan** [comment]
Same thing occurs for me on 
Version 1.69.168 Chromium: 128.0.6613.138 (Official Build) (x86_64)
MacOS
I just get an infinite loop for Clouflare CAPTCHA

[
<img width="874" alt="Screenshot 2024-10-04 at 16 26 34" src="https://github.com/user-attachments/assets/76efbfc2-e775-493d-afbd-10bbe0f2efc5">
](url)

**Comment by lawweiliang** [comment]
I found the solution. 
The issue is caused by your browser extensions.

```Turn off all your browser extensions and refresh your page.```

I get the solutions from the comments of this video. 
https://www.youtube.com/watch?v=5z0O5U8Q5dE

![Image](https://github.com/user-attachments/assets/80a37840-3066-41f7-b7ab-064ec7f7a31b)

**Comment by onchov** [comment]
Related to #27006 ?

I experienced the issue until recently and it seems to have been solved in v1.81.137.

The only affected OS was Windows 10 (I also use Brave on Linux and Android).

**Comment by candroid-man** [comment]
@onchov are you using a VPN browser extension? That was the issue for me

**Comment by onchov** [comment]
> [@onchov](https://github.com/onchov) are you using a VPN browser extension? That was the issue for me

Yes, the next time I experience issues, I'll check that out, thanks.

**Comment by vacuumdiagram** [comment]
Alas, website I want to access, I can only access via VPN.  :-/
it is, in fact, the reason that I got the Brave VPN.  I had been able to access the site using it though, for the last couple of months up  until today.

**Comment by bsclifton** [comment]
Hi folks - is this still an issue for anyone here? @vacuumdiagram @onchov @candroid-man 

cc: @ShivanKaul in case there are some tests we can recommend folks to try

**Comment by candroid-man** [comment]
It happens with certain VPN browser extensions but not all of them? Windscribe's extension works the worst in my experience and I'm not sure if that's their fault or not?

**Comment by vacuumdiagram** [comment]
I've not had it happen for a little while.  I don't have any extensions, it
was just happening with Brave by itself.  I do still find that some
websites will not load at all, or don't work properly, with the Brave VPN
enabled, but not had recent issues specifically with Cloudflare Captchas,
cheers.

**Comment by ShivanKaul** [comment]
this is very stale, we've since worked with Cloudflare to make sure this doesn't happen. please do let me know if it starts happening again. for future reference https://browser-compat.turnstile.workers.dev/ is the official test website

**Comment by mwiles** [comment]
@ShivanKaul per requested.  It appears to be happening again.  Below is my Turnstile Troubleshooter screenshot:

<img width="927" height="1065" alt="Image" src="https://github.com/user-attachments/assets/61c08ccf-7a2e-4ebe-a395-d58351ee2295" />

And the Turnstile Troubleshooter session ID 19F47360D197CB3ACA8.  

Anytime I see this Cloudflare Captcha I have to swap to a Firefox based browser (Firefox itself or Librewolf) to get past it.  To be clear,  I change nothing on Firefox based browsers such as turning off VPN or change DNS.  It immediately passes on Firefox and sometimes I don't even get the captcha.  It is not a specific site, it is every Cloudflare captcha I run into right now.

**Comment by ShivanKaul** [comment]
Gotcha. What OS and platform are you on? Please let me know if there are others also running into this.

**Comment by mwiles** [comment]
> Gotcha. What OS and platform are you on? Please let me know if there are others also running into this.

Thank you for the quick response.  OS and platform information:

CachyOS Linux: 6.18.37-1-cachyos-lts

Brave 1.92.134 (Official Build)
Chromium: 150.0.7871.63

Hyprland 0.55.4 built from branch v0.55.4 at commit a0136d8c04687bb36eb8a28eb9d1ff92aea99704 clean ([gha] Nix: update inputs).
Date: Thu Jun 11 17:10:04 2026
Tag: v0.55.4, commits: 7378

Libraries:
Hyprgraphics: built against 0.5.1, system has 0.5.1
Hyprutils: built against 0.13.1, system has 0.13.1
Hyprcursor: built against 0.1.13, system has 0.1.13
Hyprlang: built against 0.6.8, system has 0.6.8
Aquamarine: built against 0.12.1, system has 0.12.1

CPU: 12th Gen Intel(R) Core(TM) i7-12700H (12+8) @ 4.70 GHz
GPU 1: NVIDIA GeForce RTX 3080 Ti Mobile [Discrete]
GPU 2: Intel Iris Xe Graphics @ 1.40 GHz [Integrated]

Please let me know if you need any additional information.
